.github/workflows/ci.yml

```diff
@@ -26,3 +26,42 @@ jobs:
 
       - name: Clean and build
         run: ./gradlew clean build -Plog-tests
+
+  publish:
+    name: Publish
+    needs: [build]
+    if: github.event_name != 'pull_request' && ((github.ref == 'refs/heads/dss') || startsWith(github.ref, 'refs/tags/v'))
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup mill
+        uses: jodersky/setup-mill@master
+        with:
+          mill-version: 0.9.10
+
+      - name: Cache mill
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.mill
+            ~/.ivy2/cache
+            ~/.coursier/cache/v1
+            ~/.cache/coursier/v1
+            ~/AppData/Local/Coursier/Cache/v1
+            ~/Library/Caches/Coursier/v1
+          key: mill-cache-v2-${{ hashFiles('**/*.sc') }}-${{ hashFiles('.mill-version') }}
```

Comment:
I thought coursier cache supports those folders - or is it just for the mill version? https://github.com/coursier/cache-action#mill

Reply:
Not sure, I've blindly copied it from elsewhere 😅

Reply:
Nice : 36d051a